etna/ciim/utils.py

```diff
@@ -259,7 +259,22 @@ def format_link(link_html: str) -> Dict[str, str]:
 
 
 def strip_html(value: str, preserve_marks=False):
```

Comment:
**Suggestion:** We could enforce `preserve_marks` to be a keyword-only argument given it makes no sense as a positional one at all.

If you were reading `strip_html(code, True)` in isolation, it is meaningless just seeing `True` because you don't know what it means without seeing the definition of `strip_html` itself.